make-debs.sh

```diff
@@ -80,14 +80,22 @@ fi
 # b) do not sign the packages
 # c) use half of the available processors
 #
-: ${NPROC:=$(($(nproc) / 2))}
```

Comment:
I'm a little uncertain about throwing out the NPROC if it's set in the existing environment. In other scripts I just added back setting NPROC so that external scripts or users that "know better" can override the heuristics in the scripts.